server/routes/formatters.js

```diff
@@ -210,38 +210,40 @@ function createPendingTransactionsResponseFrom(pending) {
 
     return {
       total: formatBalanceOrDefault(null, totalPendingPenceAmount / 100, 'GBP'),
-      head: ['Date', 'Amount', 'Payment description', 'Prison'].map(
-        toGovUkTableCells,
-      ),
+      head: [
+        'Date',
+        'Money in',
+        'Money out',
+        'Payment description',
+        'Prison',
+      ].map(toGovUkTableCells),
       rows: holdingNotCleared
-        .map(transaction => ({
-          paymentDate: formatDateOrDefault(
-            '',
-            'd MMMM yyyy',
-            transaction.entryDate,
+        .map(
+          ({
+            entryDate,
+            postingType,
+            penceAmount,
+            currency,
+            entryDescription: paymentDescription,
+            prison,
+          }) => ({
+            paymentDate: formatDateOrDefault('', 'd MMMM yyyy', entryDate),
+            moneyIn:
+              postingType === 'CR'
+                ? formatBalanceOrDefault(null, penceAmount / 100, currency)
+                : '',
+            moneyOut:
+              postingType === 'CR'
+                ? ''
+                : formatBalanceOrDefault(null, 0 - penceAmount / 100, currency),
+            paymentDescription,
+            prison,
+          }),
+        )
+        .map(({ paymentDate, moneyIn, moneyOut, paymentDescription, prison }) =>
+          [paymentDate, moneyIn, moneyOut, paymentDescription, prison].map(
```

Comment:
`.map(transaction => Object.keys(transaction).map(`
Would work, but assumes that we are using all the keys and I'm not sure we are.... ...so not implemented.

Reply:
Also I presume the order matters too

Reply:
actually I guess you do know that you are using all the keys as you are specfically building up the object in the mapping step above

Reply:
I appreciate, its a little splitting hairs but... just re-looking over this, the secondary map is kinda pointless? why not just create the array straight away instead of creating as an object and then converting to an array afterwards?